FungibleToken.test.ts

```diff
@@ -61,6 +66,7 @@ describe("token integration", () => {
     if (proofsEnabled) {
       await FungibleToken.compile()
```

Comment:
Can these not be parallelized? (`Promise.all([ ... ])`)

Reply:
No, unfortunately that wouldn't work. Multiple provers in the same JavaScript is not supported.